### PR TITLE
Adding component_spec file with test to review accessibility of first component

### PR DESCRIPTION
**Pull request by george-bit**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,1692 @@
+{
```
Comment (tvararu) [inline package-lock.json]: Can you remove this file? It's created when the project is bundled using `npm i`. You should bundle this project with `yarn`.

File: cypress/integration/components.spec.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+const thenItShouldBeAccessible = () => {
+  cy.injectAxe();
+  cy.checkA11y(
+    null,
+    {
+      includedImpacts: ["critical"],
+    },
+    terminalLog
+  );
```
Comment (tvararu) [inline cypress/integration/components.spec.js]: I think these two + `terminalLog` could be refactored to a command in `support/commands.js`. Then we could reuse them in both specs and avoid some duplication. Could you try that? 🚢

**Review by tvararu — COMMENTED** [review]
Just a DRYing suggestion 👍

File: cypress/integration/components.spec.js (hunk 2 above)
Comment (george-bit) [reply cypress/integration/components.spec.js]: Makes sense I'll have a look

File: package-lock.json (hunk 1 above)
Comment (george-bit) [reply package-lock.json]: hmm okay i was certain that I ran yarn cypress but I will remove.

**Pull request merged**